exercises/concept/log-levels/.meta/design.md

```diff
@@ -22,3 +22,16 @@
 This exercise's prerequisites Concepts are:
 
 - `basics`: know how to define methods.
+
+## Analyzer
+
+This exercise could benefit from the following rules in the [analyzer]:
+
+- `actionable`: If the student did not reuse the implementation of the `message` and `logLevel` methods in the `reformat` method, instruct them to do so.
+- `actionable`: If the solution did not use `substring` in the methods `message` and `logLevel`, instruct them to do so.
+- `informative`: If the solution uses string concatenation in the `reformat` method, suggest to the student that it can be simplified using `String.format`.
```

Comment:
I'm not sure if we should be suggesting using `String.format` instead of string concatenation. There could be other reasons for preferring string concatenation, like performance (see [two fer mentoring notes](https://github.com/exercism/website-copy/blob/35199a4447adcce69204ad296d8e29ea3ca792a8/tracks/java/exercises/two-fer/mentoring.md?plain=1#L35)).

Reply:
Finally @kahgoh I took your advise but invert how the recommendation works!